src/modules/settlement/breezekiln/breezekiln.gd

```diff
@@ -0,0 +1,20 @@
+extends Node2D
+
+var _settlement_data: SettlementData
+
+
+func start_fire_fighters() -> void:
+	var minigame_data = MinigameData.new()
+	minigame_data.display_name = "fire_fighters"
+	minigame_data.minigame_scene = load("res://modules/minigame/fire_fighters/fire_fighters.tscn")
+	_debug_force_settlement_data()
```

Comment:
This special function is for debugging. Normally during game play, the player will already have a settlement they are in as they navigate the world.